r/vignettes/developers/workflow.Rmd

```diff
@@ -4,6 +4,22 @@
 knitr::opts_chunk$set(error = TRUE, eval = FALSE)
 ```
 
+The Arrow R package uses several additional development tools:
+
+* `lintr` for code analysis
+  - for the time being, the R package uses a custom version of lintr - `jonkeane/lintr@arrow-branch`
+* `styler` for code styling
+* `pkgdown` for building the website
+* `roxygen2` for documenting the package
```

Comment:
I really like how you've mentioned the packages up-front here; how about also adding links to their pkgdown sites when you click on their names?

Reply:
That sounds like a great idea. We might need to that in a separate PR as I don't think we are going to merge this PR anytime soon.

Reply:
Done, with the exception of {lintr} which does not have a pkgdown site - I linked to the GitHub.